### PR TITLE
Fixed issue #6558

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#6558

## Description
Fixed the CSS of the cross icon on booking product

### CODE REVIEW

File: packages/Webkul/BookingProduct/src/Resources/assets/sass/velocity.scss [hunk 1]
```diff
@@ -117,6 +117,16 @@
         .control-group-container {
             width: 100%;
             float: left;
+            
+            span {
+                position:relative;
```
Comment (devansh-webkul) [inline packages/Webkul/BookingProduct/src/Resources/assets/sass/velocity.scss:122]: Add one space after the colon.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/BookingProduct/src/Resources/assets/sass/velocity.scss (hunk 1 above)
Comment (akathuria289) [reply packages/Webkul/BookingProduct/src/Resources/assets/sass/velocity.scss]: Both Added

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
**Not working** for Arabic locale, please find the screenshot [Here](https://prnt.sc/H7qcN4ZzsQtg)

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Fixed

File: packages/Webkul/BookingProduct/src/Resources/views/shop/velocity/products/view/booking/slots.blade.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
             <label class="label-style required">{{ $title ?? __('bookingproduct::app.shop.products.book-an-appointment') }}</label>
 
             <div class="control-group-container">
-                <div class="form-group date" :class="[errors.has('booking[date]') ? 'has-error' : '']">
+                <div class="form-group date {{$product->locale == 'ar' ? 'rtl' : ''}}" :class="[errors.has('booking[date]') ? 'has-error' : '']">
```
Comment (devansh-webkul) [inline packages/Webkul/BookingProduct/src/Resources/views/shop/velocity/products/view/booking/slots.blade.php:10]: No need for a case here, as the body tag already has rtl. Handle it from scss in rtl file.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by akathuria289** [comment]
@devansh-webkul Done

**Comment by Sanjaybhattwebkul** [comment]
New pr created for the same issue : https://github.com/bagisto/bagisto/pull/6595

**Pull request closed without merging**